specification/security/resource-manager/Microsoft.Security/stable/2022-05-01/settings.json

```diff
@@ -0,0 +1,306 @@
+    "DataExportSettings": {
+      "type": "object",
+      "description": "Represents a data export setting",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Data export setting data",
+          "$ref": "#/definitions/DataExportSettingProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Setting"
+        }
+      ],
+      "x-ms-discriminator-value": "DataExportSettings"
+    },
+    "AlertSyncSettings": {
+      "type": "object",
+      "description": "Represents an alert sync setting",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Alert sync setting data",
+          "$ref": "#/definitions/AlertSyncSettingProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Setting"
+        }
+      ],
+      "x-ms-discriminator-value": "AlertSyncSettings"
+    },
+    "Setting": {
+      "type": "object",
+      "description": "The kind of the security setting",
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "the kind of the settings string",
+          "enum": [
+            "DataExportSettings",
+            "AlertSuppressionSetting",
+            "AlertSyncSettings"
+          ],
+          "x-ms-enum": {
+            "name": "SettingKind",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "DataExportSettings"
+              },
+              {
+                "value": "AlertSuppressionSetting"
+              },
+              {
+                "value": "AlertSyncSettings"
+              }
+            ]
+          }
+        }
+      },
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "DataExportSettingProperties": {
+      "type": "object",
+      "description": "The data export setting properties",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Is the data export setting enabled"
+        }
+      },
+      "required": [
+        "enabled"
+      ]
+    },
+    "AlertSyncSettingProperties": {
+      "type": "object",
+      "description": "The alert sync setting properties",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Is the alert sync setting enabled"
+        }
+      },
+      "required": [
+        "enabled"
+      ]
+    }
+  },
+  "parameters": {
+    "SettingName": {
+      "name": "settingName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the setting",
+      "enum": [
+        "MCAS",
+        "WDATP",
+        "WDATP_EXCLUDE_LINUX_PUBLIC_PREVIEW",
+        "WDATP_UNIFIED_SOLUTION",
```

Comment:
Is this the only change ? renaming the enum value ?This would be a breaking change , please get approval if that is required.

Reply:
the change is not renaming an enum value, it's adding a new value to it WDATP_UNIFIED_SOLUTION.
please notice that i made a very similar change on July 2021 (added the value WDATP_EXCLUDE_LINUX_PUBLIC_PREVIEW to the enum) that was accepted #15093